profiling/src/lib.rs

```diff
@@ -83,6 +83,21 @@ lazy_static! {
             .expect("Reflection's zend_module_entry to be found and contain a valid string")
     };
 
+    static ref PHP_VERSION_STRUCTURED: Option<(u32, u32, u32)> = {
+        let mut iter = PHP_VERSION.split('.').fuse();
+
+        let major = iter.next()?.parse().ok()?;
+        let minor = iter.next()?.parse().ok()?;
+        let patch = iter
+            .next()?
+            .split(|c: char| !c.is_numeric())
+            .next()?
+            .parse()
+            .ok()?;
+
+        Some((major, minor, patch))
+    };
+
```

Comment:
I am wondering if we should try and go for something `PHP_VERSION_ID` like, so that we can just call `php_version_id()` from PHP 8.3 onwards?
https://github.com/php/php-src/pull/11875

Reply:
Ah yes, this looks nicer 😄